app/policies/organizations/adopter_fosterer/match_policy.rb

```diff
@@ -1,12 +1,17 @@
 module Organizations
   module AdopterFosterer
     class MatchPolicy < ApplicationPolicy
+      authorize :page
       relation_scope do |relation|
         relation.where(person_id: person.id)
       end
 
       def index?
-        permission?(:view_adopter_foster_matches)
+        if page == "fostered_pets"
+          person.active_in_group?(:fosterer) && permission?(:view_foster_adopter_matches)
```

Comment:
`active_in_group` is already checked in `authorizable.rb`

```suggestion
          permission?(:view_foster_adopter_matches)
```